mlflow/gateway/config.py

```diff
@@ -274,6 +282,23 @@ def validate_model(cls, model):
                 )
         return model
 
+    @root_validator(skip_on_failure=True)
```

Comment:
Yes, this is deprecated in pydantic 2.x, but so are our other validators. We'll need to move to @model_validator @field_validator implementations eventually when we sunset pydantic 1.x